### ISSUE TITLE
Crash when building between blockers

### ISSUE THREAD

**Issue by geoo89** [SimonN/lix-unstable#1]
I encounter a crash of the game in version 7.2.0. dmd version is v2.073.1. System: Ubuntu 14.04 LTS 64 bit.

Setup: put two blockers close to each other (such that the blocker fields overlap slightly, or that there is a very small gap between the blocker fields). Assign a builder between the two blockers. After a few bricks, a crash occurs (game closes).

Error trace:

```
core.exception.AssertError@src/lix/skill/builder.d(68): Assertion failure
----------------
??:? _d_assert [0x8caa44]
??:? void lix.skill.builder.__assert(int) [0x88f5d9]
src/lix/skill/builder.d:68 void lix.skill.builder.BrickCounter.buildBrick() [0x88e660]
src/lix/skill/builder.d:61 void lix.skill.builder.BrickCounter.perform() [0x88e5e0]
src/lix/perform.d:16 void lix.perform.performUseGadgets(lix.lixxie.Lixxie) [0x88c0f8]
src/lix/lixxie.d:514 void lix.lixxie.Lixxie.perform(lix.fields.OutsideWorld*) [0x88bad4]
src/game/model/model.d:261 _D4game5model5model9GameModel13updateLixxiesMFZ15performUnmarkedMFE3net2ac9PhyuOrderZ9__lambda2MFC4game5tribe5TribexiC3lix6lixxie6LixxieZv [0x832024]
src/game/model/model.d:223 _D4game5model5model9GameModel13updateLixxiesMFZ10foreachLixMFDFC4game5tribe5TribexiC3lix6lixxie6LixxieZvZv [0x831d35]
src/game/model/model.d:257 _D4game5model5model9GameModel13updateLixxiesMFZ15performUnmarkedMFE3net2ac9PhyuOrderZv [0x831f9b]
src/game/model/model.d:274 void game.model.model.GameModel.updateLixxies() [0x831c5b]
src/game/model/model.d:90 void game.model.model.GameModel.advance() [0x8310e4]
src/game/model/nurse.d:186 void game.model.nurse.Nurse.updateOnce() [0x833777]
src/game/model/nurse.d:125 void game.model.nurse.Nurse.updateTo(const(net.phyu.Phyu), const(game.model.cache.DuringTurbo)) [0x832f55]
src/game/core/speed.d:26 void game.core.speed.updatePhysicsAccordingToSpeedButtons(game.core.game.Game).upd(const(int), const(game.model.cache.DuringTurbo)) [0x827d5f]
src/game/core/speed.d:74 void game.core.speed.updatePhysicsAccordingToSpeedButtons(game.core.game.Game) [0x827cd5]
src/game/core/calc.d:25 void game.core.calc.implGameCalc(game.core.game.Game).noninputCalc() [0x823581]
src/game/core/calc.d:37 void game.core.calc.implGameCalc(game.core.game.Game) [0x8234f9]
src/game/core/game.d:194 void game.core.game.Game.calc() [0x825cc5]
src/basics/mainloop.d:251 void basics.mainloop.MainLoop.calc() [0x74e392]
src/basics/mainloop.d:85 void basics.mainloop.MainLoop.mainLoop() [0x74da1f]
src/main.d:37 int main.main(immutable(char)[][]).__lambda2() [0x895f4c]
../../../home/georg/.dub/packages/allegro-4.0.1_5.2.0/allegro5/system.d:45 extern (C) int allegro5.system.al_run_allegro(scope int delegate()).main_runner(int, char**) [0x8c635a]
../../../home/georg/.dub/packages/allegro-4.0.1_5.2.0/allegro5/system.d:62 int allegro5.system.al_run_allegro(scope int delegate()) [0x8c6331]
src/main.d:35 _Dmain [0x895ee3]
??:? _D2rt6dmain211_d_run_mainUiPPaPUAAaZiZ6runAllMFZ9__lambda1MFZv [0x8cf22f]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x8cf157]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll() [0x8cf1d4]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x8cf157]
??:? _d_run_main [0x8cf0c3]
??:? main [0x896011]
??:? __libc_start_main [0xb2731f44]
Error executing command run:
Program exited with code 1
```

**Comment by SimonN** [comment]
`assert (bricksLeft > 0)` fails, looks like a serious logic error on my side.

I've tried (two blockers with overlapping blocker fields, then build with a thrid lix in the middle). The game didn't crash, but the builder ran out of bricks after placing 1 or 2 bricks.

**Comment by SimonN** [comment]
Should be fixed in 0.7.6, branch exp.
